sdv/metadata/anonymization.py

```diff
@@ -57,7 +58,9 @@ def is_faker_function(function_name):
         True if the ``function_name`` is know to ``Faker``, otherwise False.
     """
     try:
+        warnings.filterwarnings('ignore', module='faker')
         getattr(Faker(AVAILABLE_LOCALES), function_name)
+        warnings.resetwarnings()
```

Comment:
I think that we should use [catch_warnings](https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings) instead.
```python
        with warnings.catch_warnings():
            warnings.filterwarnings('ignore', module='faker')
            getattr(Faker(AVAILABLE_LOCALES), function_name)
```
If an user has configured something else to the warnings this `warnings.resetwarnings()` may reset that configuration.